microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools.md

```diff
@@ -111,7 +111,7 @@ For more information, check out [Elements for Microsoft Edge Visual Studio Code
 
 #### Debugger for Microsoft Edge
 
-With the [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) Visual Studio Code extension, debug JavaScript running in Microsoft Edge directly from Visual Studio Code.
+With the [Microsoft Edge Tools for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools) Visual Studio Code extension, debug JavaScript running in Microsoft Edge directly from Visual Studio Code.
```

Comment:
@codepo8 - https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge has become 404.  Should this h4 section be removed? Rendered public What's New entry has the 404 link:  https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools#debugger-for-microsoft-edge

Is this page valid? - https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/debugger-for-edge

I'm working in an "Old What's New processing" PR https://github.com/MicrosoftDocs/edge-developer/pull/1881 where I can do more elaborate fixes.  The present PR could just delete the 404 link or wrap with backticks to disable it.

Reply:
@codepo8 - I'm closing this Comment, to approve this PR, to fix the 404 asap.  We can continue discussing redesign in my "Old What's New" processing PR 1881, re: what to do with old mentions of Debugger Extension.  Probably all mentions of the Debugger Extension are in old What's New items.